src/extensions/starter_dashlet/Tools.tsx

```diff
@@ -130,17 +134,21 @@ class Starter extends ComponentEx<IStarterProps, IWelcomeScreenState> {
       const discoveredGame = discoveredGames[gameMode];
       content = (
         <Media id='starter-dashlet'>
-          <Media.Left>
-            {this.renderGameIcon(game, discoveredGame)}
-            {this.renderEditToolDialog()}
-          </Media.Left>
           <Media.Body>
-            {this.renderToolIcons(game, discoveredGame)}
+            <FlexLayout type='column'>
+              <FlexLayout type='row' className='starter-dashlet-tools-header'>
+                <h1>{t('Tools')}</h1>
+                <Toggle
+                  checked={addToTitleBar}
+                  onToggle={this.onToggle}
+                >
+                  {t('Enable toolbar')}
```

Comment:
So afaict the "toolbar" is still provided by a "dynamic" extension whereas this is part of the base application. That's really not how extensions are supposed to interact.